doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -441,13 +414,9 @@ int main(int argc, char** argv)
   move_group_interface_tutorial.setupVisualization();
   move_group_interface_tutorial.printBasicInfo();
 
-  // Create the plan we will be using
-  moveit::planning_interface::MoveGroupInterface::Plan plan;
-  // This is a vector of locations for moveing to multiple targets.
-  std::vector<geometry_msgs::Pose> target_poses;
-
   // Plan and move to Joint space goal
-  move_group_interface_tutorial.publishToRViz("Joint Space Goal");
+  std::string publish_message = "Joint Space Goal";
```

Comment:
you could simply pass this in directly to publishToRviz rather than use this intermediate var

Reply:
If I am publishing the same message to RViz 3-4 times I thought it made it easier to have an intermediate variable.